datamodel/initdb.d/03_dcsa_im_v3_0.sql

```diff
@@ -264,6 +264,8 @@ CREATE TABLE dcsa_im_v3_0.booking (
     incoterms varchar(3) NULL REFERENCES dcsa_im_v3_0.incoterms(incoterms_code),
     invoice_payable_at varchar(100) NULL REFERENCES dcsa_im_v3_0.location(id),
     expected_departure_date timestamp NULL,
+    expected_arrival_date_start timestamp NULL,
+    expected_arrival_date_end timestamp NULL,
```

Comment:
Should we have a `CHECK ((expected_arrival_date_start IS NULL AND expected_arrival_date_end IS NULL) OR expected_arrival_date_start < expected_arrival_date_end)` to ensure that start is less than end date when both are provided?

Reply:
Good point - added